### PR TITLE
Add bash startup script and info on how to run it

### PR DESCRIPTION
**Pull request by zingo**
Signed-off-by: Zingo Andersen <zingo@zingo.org>

### CODE REVIEW

**Comment by zingo** [comment]
Hi tried your project and got it running without powershell (as I don't have it installed)
I tried to add the info I needed to do to the README.md so it would be easier for the next person to use it :) It's totally ok if you don't want this or want me to clean it up, just let me know if that is the case. I'm new to this node stuff so my readme changes could be a bit off depending on how you are supposed to do it :)

**Comment by instance-id** [comment]
Ah yes, good call. I use PowerShell for a lot of my scripts and forgot to include a bash version. Don't feel bad about the node stuff, I don't really know it that well either. Just enough to make this, lol. What you added to the readme sums it up pretty well.

I never really planned for anyone else to use it, so a lot is quite specific to my needs, but I might go in and add a few config bools to allow enabling or disabling features (such as the MQTT stuff) so as not to cause any unnecessary errors or possible crashing if things are not used/filled out, such as the .env file for the MQTT server connection info. I will also add in dummy example of the .env in case someone does want to actually use it. At that point I can go back to the readme and try and give some more details on the features and how to use them.

File: README.md [hunk 1]
```diff
@@ -21,3 +21,39 @@ It may be a bit diffifcult to figure out at first, but this includes an MQTT cli
 See the below repo that the base library is capable of.  
 Powered by Loupedeck Node library
 Unofficial Node.js API for [Loupedeck Live](https://loupedeck.com/products/loupedeck-live/) & [Loupedeck Live S](https://loupedeck.com/products/loupedeck-live-s/) controllers.
+
+#### Dependencies
+Make sure you have the following installed in your system:
+
+node
```
Comment (instance-id) [inline README.md:28]: An extra "node" is in the list of deps.

Comment (zingo) [reply README.md:28]: Good catch! Ill fix, I also fixt a problem with the logfile

**Review by instance-id — APPROVED** [review]

**Comment by zingo** [comment]
> I never really planned for anyone else to use it, so a lot is quite specific to my needs, but I might go in and add a few config bools to allow enabling or disabling features (such as the MQTT stuff) so as not to cause any unnecessary errors or possible crashing if things are not used/filled out, such as the .env file for the MQTT server connection info. I will also add in dummy example of the .env in case someone does want to actually use it. At that point I can go back to the readme and try and give some more details on the features and how to use them.

No worries about that, I assume you as me treat this a hobby free time codeing and the code envolves depending on the current need :) 

If it's ok Ill start to configure some buttons on my side and I might change some code during the process, Ill try to keep your stuff in a good shape in my fork so we can pull between our  forks. I'm happy to share my changes and you can decide if you want to bother or not with them depending on the change or your time if/when they happen :) e.g. my mindset is that I don't expect you to do anything but are happy to share my stuff back when it makes sense :)

**Pull request merged**